ext/build_defaults.yaml

```diff
@@ -25,15 +25,14 @@ foss_platforms:
   - windows-2012-x86
   - windows-2012-x64
 pe_platforms:
-  - aix-6.1-power
   - aix-7.1-power
```

Comment:
It's not clear to me why 7.2 is missing, but that seems like a different issue?

Reply:
Historically, AIX 7.1 and 7.2 were using AIX 6.1 packages. Now with the removal of AIX 6.1 both 7.1 and 7.2 will use 7.1 packages.